lib/compiler/build_runner.zig

```diff
@@ -66,6 +66,12 @@ pub fn main() !void {
         .handle = try std.fs.cwd().makeOpenPath(global_cache_root, .{}),
     };
 
+    // Match the behavior of `cmdBuild()`.
```

Comment:
Perhaps it would be best to have a `std.zig.default_build_runner_target_query` or similar to prevent these from getting out of sync.